src/types/cls.ts

```diff
@@ -81,14 +81,14 @@ export interface CLSMetricWithAttribution extends CLSMetric {
 /**
  * A CLS-specific version of the ReportCallback function.
  */
-export interface CLSReportCallback extends ReportCallback {
+export interface CLSReportCallback extends Omit<ReportCallback, 'metric'> {
   (metric: CLSMetric): void;
 }
 
 /**
  * A CLS-specific version of the ReportCallback function with attribution.
  */
 export interface CLSReportCallbackWithAttribution
-  extends ReportCallbackWithAttribution {
+  extends Omit<ReportCallbackWithAttribution, 'metric'> {
```

Comment:
Same question here, as well as in all the other report callback variants...

Given that this is just a function and not a type with a number of different properties on it, it's not clear to me that the `Omit<>` is providing any value (e.g. making the code more DRY or anything like that).

Reply:
Ditto